helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/model/AssignableNode.java

```diff
@@ -222,6 +222,12 @@ public int getAssignedReplicaCount() {
     return _currentCapacityMap;
   }
 
+
+  //TODO: will implement by merging other changes
+  public Map<String, Integer> getCapacityUsage() {
```

Comment:
Let's don't make it public since it is not supporting the feature.
I would prefer the model class to be clean and no extra method, as long as it is possible.